### ISSUE TITLE
phidata-lite

### ISSUE THREAD

**Issue by jacobweiss2305** [phidatahq/phidata#1307]
Hey Team, 

The current phidata pypi package is starting to get bulky. I was wondering if we can collaborate on a phidate-lite version that trims out:

- IaC which include aws, workspace, 
- Assistant (deprecated in 3.0.0)
- Tools
- Vectordb

My thought is that we modularize the packages so that devs can install only the necessary tools

I am seeing this as a common design pattern with langchain, LlamaIndex, crewai

Thanks,
Jacob

**Comment by ashpreetbedi** [comment]
good call, we'll probably pull out the tools, Iac and deprecate assistants in v3.
